spec/fixtures/constants.rb

```diff
@@ -575,3 +575,17 @@
                          %d
 1 warning generated.
 )
+
+
+################################################################################
+# INFORMATIVE
+################################################################################
+
+SAMPLE_PLUGIN_INFO = %q(2015-04-23 10:20:09.018 xcodebuild[77828:9774464] [MT] PluginLoading: Required plug-in compatibility UUID 9F75337B-21B4-4ADC-B558-F9CADF7073A7 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/AllMissingXcodeFeatures.xcplugin' not present in DVTPlugInCompatibilityUUIDs)
+
+SAMPLE_PLUGIN_INFO_TRIMMED = %q([MT] PluginLoading: Required plug-in compatibility UUID 9F75337B-21B4-4ADC-B558-F9CADF7073A7 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/AllMissingXcodeFeatures.xcplugin' not present in DVTPlugInCompatibilityUUIDs)
```

Comment:
Line is too long. [285/80]<br>Use `%q` only for strings that contain both single quotes and double quotes.